src/App.tsx

```diff
@@ -1,10 +1,19 @@
-import React from "react";
-import { Routes, Route } from "react-router-dom";
+import React, { useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import { Routes, Route, useSearchParams } from "react-router-dom";
 
 import AboutPage from "./pages/About";
 import HomePage from "./pages/Home";
 
 const App: React.FunctionComponent = () => {
+  const { i18n } = useTranslation();
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  useEffect(() => {
+    searchParams.set("lng", i18n.resolvedLanguage);
```

Comment:
```suggestion
    searchParams.set("lang", i18n.resolvedLanguage);
```
`lang` or `locale` or `hl`
never use strangely shortened names, because if something looks clear for you, can be not obvious for other